src/main/java/upbrella/be/rent/service/ConditionReportService.java

```diff
@@ -0,0 +1,28 @@
+package upbrella.be.rent.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import upbrella.be.rent.dto.response.ConditionReportResponse;
+import upbrella.be.rent.repository.ConditionReportRepository;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class ConditionReportService {
+
+    private final ConditionReportRepository conditionReportRepository;
+
+    @Transactional
+    public List<ConditionReportResponse> findAllConditionReport() {
+        return conditionReportRepository.findAll().stream()
+                .map(conditionReport -> ConditionReportResponse.builder()
+                        .id(conditionReport.getId())
+                        .umbrellaId(conditionReport.getHistory().getId())
+                        .content(conditionReport.getContent())
+                        .build())
+                .collect(Collectors.toList());
```

Comment:
여기서도 매핑 로직을 객체로 분리해도 좋을것같아요
```suggestion
                .map(ConditionReportReponse::from)
                .collect(Collectors.toUnmodifiableList());
```

Reply:
좋은 것같아요. 수정하겠습니다~!